lib/result-cache-json.js

```diff
@@ -41,7 +49,7 @@ function replacer(_key, value) {
   }
 
   if (typeof value !== 'number' || Number.isFinite(value)) {
-    return value;
+    return value ?? null;
```

Comment:
This was unintentional.
Well, I intended it initially, but fixed some other type errors, so this should be unnecessary.